hassio/src/components/supervisor-snapshot-content.ts

```diff
@@ -0,0 +1,384 @@
+            </div>
+            <ha-formfield
+              .label=${this.supervisor.localize("snapshot.full_snapshot")}
+            >
+              <ha-radio
+                @change=${this._handleRadioValueChanged}
+                value="full"
+                name="snapshotType"
+                .checked=${this.snapshotType === "full"}
+              >
+              </ha-radio>
+            </ha-formfield>
+            <ha-formfield
+              .label=${this.supervisor!.localize("snapshot.partial_snapshot")}
+            >
+              <ha-radio
+                @change=${this._handleRadioValueChanged}
+                value="partial"
+                name="snapshotType"
+                .checked=${this.snapshotType === "partial"}
+              >
+              </ha-radio>
+            </ha-formfield>
+          </div>`
+        : ""}
+      ${this.snapshot && this.snapshotType === "partial"
+        ? html`
+            ${this.snapshot.homeassistant
+              ? html`<div
+                  class="checkbox-row"
+                  @click=${() => {
+                    this.homeAssistant = !this.homeAssistant;
+                  }}
+                >
+                  <ha-checkbox .checked=${this.homeAssistant}> </ha-checkbox>
+                  <ha-svg-icon .path=${mdiHomeAssistant} class="icon">
+                  </ha-svg-icon>
+                  <span class="item-name">Home Assistant</span>
+                  <span class="item-version">
+                    (${this.snapshot.homeassistant})
+                  </span>
+                </div>`
+              : ""}
+          `
+        : ""}
+      ${this.snapshotType === "partial"
+        ? html`
+            ${this.folders?.length
+              ? html`
+                  <div
+                    class="checkbox-row"
+                    @click=${() =>
+                      this._toggleSection("folders", this.folders!)}
+                  >
+                    <ha-checkbox
+                      ?indeterminate=${this._sectionIndeterminate(this.folders)}
+                      .checked=${this._sectionCheked(this.folders)}
+                    >
+                    </ha-checkbox>
+                    <ha-svg-icon .path=${mdiFolder} class="icon"></ha-svg-icon>
+                    ${this.supervisor.localize("snapshot.folders")}
+                  </div>
+                  <div class="section-content">
+                    ${this.folders.map(
+                      (item) => html`
+                        <div
+                          class="checkbox-row"
+                          @click=${() => this._updateFolder(item)}
+                        >
+                          <ha-checkbox .checked=${item.checked}> </ha-checkbox>
+                          <ha-svg-icon .path=${mdiFolder} class="icon">
+                          </ha-svg-icon>
+                          ${item.name}
+                        </div>
```

Comment:
Not sure, but I think this should do the same?
```suggestion
                        <ha-formfield>
                          <ha-checkbox .item=${item} .checked=${item.checked} @click=${this._updateFolder}></ha-checkbox>
                          <ha-svg-icon .path=${mdiFolder} class="icon">
                          </ha-svg-icon>
                          ${item.name}
                        </ha-formfield>
```

Reply:
More or less, a little different styling, but not possible for addon, and wanted to be consistent

Reply:
Why not possible for addon?

Reply:
It needs extra style handling for the icons
I guess it can be done with inline styling 🤷

Reply:
I'll check it tomorrow... I think it should just be possible...

Reply:
nevermind... this will work, I was trying to use .label before